src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -87,6 +88,12 @@
             :start-flow?    start-flow?
             :flow-id        :wallet-flow}]]]}))
 
+(rf/reg-event-fx
+ :wallet/edit-token-to-send
```

Comment:
why do we need a new event for this? can we not re-use the other one? 🤔

Reply:
I think we can use the event `wallet/send-select-token` and dispatch `:hide-bottom-sheet` separately

Reply:
the `:wallet/send-select-token` event is navigating within the wizard, I'm not sure that even can be safetly reused
cc: @mmilad75 wdyt?

But I agree I'm missing `:wallet/clean-suggested-routes` in this event, so if the event is not reused, I must also dispatch this one missed.

Reply:
> But I agree I'm missing :wallet/clean-suggested-routes in this event, so if the event is not reused, I must also dispatch this one missed.

Maybe we should re-fetch suggested routes if there is a valid amount for the token instead of cleaning? Not sure, probably it is a design decision, but I think it would be nice for the user to re calculate routes based on the new selected token and avoid the user to re-enter the amount.